server.go

```diff
@@ -231,25 +233,16 @@ func (s *ExtensionManagerServer) Call(registry string, item string, request osqu
 func (s *ExtensionManagerServer) Shutdown() error {
 	s.mutex.Lock()
 	defer s.mutex.Unlock()
-	defer func() {
-		s.server = nil
-	}()
 	if s.server != nil {
-		err := s.server.Stop()
-		if err != nil {
-			return errors.Wrap(err, "stopping server")
-		}
-	}
-
-	defer func() {
-		s.transport = nil
-	}()
-	if s.transport != nil {
-		err := s.transport.Close()
```

Comment:
Note I discovered that the Thrift server `Stop` method already closes the transport, so this was just causing problems.